Doc/library/dis.rst

```diff
@@ -1725,6 +1713,18 @@ but are replaced by real opcodes or removed before bytecode is generated.
    Undirected relative jump instructions which are replaced by their
    directed (forward/backward) counterparts by the assembler.
 
+.. opcode:: LOAD_CLOSURE (i)
+
+   Pushes a reference to the cell contained in slot ``i`` of the "fast locals"
+   storage.
+
+   Note that ``LOAD_CLOSURE`` is replaced with ``LOAD_FAST`` in the assembler.
+   It exists to keep bytecode a little more readable.
```

Comment:
I don't think this sentence is accurate anymore (if it ever was). `LOAD_CLOSURE` still exists in order to keep `fix_cell_offsets` working. Pseudo-instructions can't make bytecode more readable, since they don't exist in bytecode.

I would just remove this sentence.